spec/jobs/central_mail/submit_form4142_job_spec.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'sidekiq/testing'
```

Comment:
redundant. 
`require 'sidekiq/testing'
Sidekiq::Testing.fake!` is in rails_helper